torchaudio/csrc/register.cpp

```diff
@@ -7,19 +7,33 @@
 namespace torchaudio {
 namespace {
 
+////////////////////////////////////////////////////////////////////////////////
+// typedefs.h
+////////////////////////////////////////////////////////////////////////////////
 static auto registerSignalInfo =
     torch::class_<SignalInfo>("torchaudio", "SignalInfo")
         .def(torch::init<int64_t, int64_t, int64_t>())
         .def("get_sample_rate", &SignalInfo::getSampleRate)
         .def("get_num_channels", &SignalInfo::getNumChannels)
         .def("get_num_frames", &SignalInfo::getNumFrames);
 
+////////////////////////////////////////////////////////////////////////////////
+// sox_io.h
+////////////////////////////////////////////////////////////////////////////////
 static auto registerGetInfo = torch::RegisterOperators().op(
     torch::RegisterOperators::options()
         .schema(
             "torchaudio::sox_io_get_info(str path) -> __torch__.torch.classes.torchaudio.SignalInfo info")
         .catchAllKernel<decltype(sox_io::get_info), &sox_io::get_info>());
 
+static auto registerLoadAudioFile = torch::RegisterOperators().op(
```

Comment:
nit: should there be a comment before, as introduced above?

Reply:
such as?

Reply:
```
////////////////////////////////////////////////////////////////////////////////
// sox_io.h
////////////////////////////////////////////////////////////////////////////////
```
but then that's redundant :) Alright, let's keep things as is.